department-of-veterans-affairs/caseflow
# Nemo/Refactored SignAndCertify.

Connects to #2583 
Although the ticket was just for no default radio button value, clearing-error-message-on-user-input feature similar what we have on other pages (ConfirmCaseDetails, ConfirmHearing) was added as well. 

To test (describing gif below):
Once page loads for the first time, Title radio option should have no default value. 
Press continue and expect to see error messages. 
Putting input should clear errors right away. 
Reloading the page, or going back from Success page should load previously entered values. 
Once on success page, make sure correct values are stored in db. 

![title](https://user-images.githubusercontent.com/4960030/28921638-c2ceb81c-7824-11e7-8c35-5e6859bb0a9a.gif)
